packages/component/src/ScreenReaderActivity.js

```diff
@@ -53,9 +53,11 @@ const ScreenReaderActivity = ({ activity }) => {
   const contentTypeMarkdown = textFormatToContentType(textFormat) === 'text/markdown';
   const displayText = messageBackDisplayText || text;
 
-  const attachmentForScreenReaderRenderers = attachments
-    .map(attachment => createAttachmentForScreenReaderRenderer({ activity, attachment }))
-    .filter(render => render);
+  const attachmentForScreenReaderRenderers = renderAttachments
```

Comment:
can you explain a scenario where user would want to skip rendering attachments in SRR?

Reply:
```js
// When "renderAttachments" is false, we will not render the content of attachments.
// That means, it will only render "2 attachments", instead of "image attachment".
// This is used in the visual transcript, where we render "Press ENTER to interact."
```